src/util/raii/CStringWrapper.cpp

```diff
@@ -0,0 +1,35 @@
+#include "util/raii/CStringWrapper.h"
+
+#include <utility>
+
+#include <glib.h>
+
+namespace xoj::util {
+inline namespace raii {
+CString::CString(char* s): data(s) {}
```

Comment:
This feels error-prone to me because it takes ownership. I would make this a static method like `CString::steal()` or `CString::take()` or some other explicit name, or even suggest renaming the class to `UniqueString` or something.

Reply:
Ok. I changed to the overly causious `static OwnedCString::assumeOwnership(char* s)`.